data-runtime/src/main/java/io/micronaut/data/runtime/operations/internal/sql/DefaultSqlPreparedQuery.java

```diff
@@ -149,6 +149,12 @@ public void bindParameters(Binder binder, InvocationContext<?, ?> invocationCont
     private int getQueryParameterValueSize(QueryParameterBinding parameter) {
         int parameterIndex = parameter.getParameterIndex();
         if (parameterIndex == -1) {
+            // If parameter (in criteria for example) is multivalued then actual parameter value size is
```

Comment:
I would avoid comments like this. Maybe `sizeOf` can have a better explanation of what it's doing. Putting issues number everywhere would make code a bit too verbose, and future changes might not be up to date.

Reply:
Yes, I didn't like it either but thought could be useful to other maintainers. But, it can be seen in git history.

Reply:
Done